content/docs/addons-shallow-renderer.md

```diff
@@ -1,68 +1,69 @@
 ---
 id: shallow-renderer
-title: Shallow Renderer
+title: Renderização superficial (Shallow Renderer)
 permalink: docs/shallow-renderer.html
 layout: docs
 category: Reference
 ---
 
-**Importing**
+**Importando**
 
 ```javascript
 import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
-var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
+var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 com npm
 ```
 
-## Overview {#overview}
+## Visão geral {#overview}
 
-When writing unit tests for React, shallow rendering can be helpful. Shallow rendering lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+Ao escrever testes unitários para o React, a renderização superficial pode ser útil. A renderização superficial o deixa renderizar um componente a um "nível simples de profundidade" e afirmar fatos sobre o que este método retorna, sem se preocupar sobre o comportamento dos componentes filhos, os quais não são instanciados ou renderizados. Isto não requer o DOM.
```

Comment:
```suggestion
Ao escrever testes unitários para o React, a renderização superficial pode ser útil. A renderização superficial nos permite renderizar um componente a um "nível simples de profundidade" e determinar fatos sobre o que a renderização retorna, sem se preocupar sobre o comportamento dos componentes filhos, os quais não são instanciados ou renderizados. Isto não requer o DOM.
```

Reply:
That's a hard one, thanks for your time! I got a bit confused by reading the _"o deixa"_, what do you think we use _"nos permite"_ here? Also assert probably means more of _"determinar"_ instead of _"afirmar"_.

I've made a comment [here](https://github.com/reactjs/pt-BR.reactjs.org/pull/50#issuecomment-462089153). Maybe we can discuss it and put it into the glossary afterwards.

Reply:
Hey @tibuurcio thanks for your feedback, I think "determinar" is better too.

Reply:
Opa @henriquejensen, show! Você pode aceitar as mudanças diretamente por aqui pela interface do github, aí ele substituia e faz commit automático. Depois disso a gente faz o merge do PR, valeu 👍